src/python/Utils/ProcessStats.py

```diff
@@ -83,7 +83,7 @@ def threadStack():
     tdict = {}
     id2name = dict([(th.ident, th.name) for th in threading.enumerate()])
     threads = []
-    for tid, stack in sys._current_frames().items():
+    for tid, stack in list(sys._current_frames().items()):
```

Comment:
can't cast to list here

Reply:
In this PR I did not change anything by hand. Every change was made automatically by futurize.

Why do you suggest that this can not be done?

The [future doc](http://python-future.org/compatible_idioms.html#dict-keys-values-items-as-a-list) have some examples that look similar to this. Nonetheless, this does not seem to be efficient at all.

As a matter of fact, it seems that future has a way to select the best memory-efficient in both py2 and py3. See
* http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items
* http://python-future.org/what_else.html#dict
* https://stackoverflow.com/a/10190228